_includes/ome2021_program_agenda.html

```diff
@@ -64,7 +64,21 @@ <h5>{{ session.name }}</h5>
     {% if subevent.description %}{{ subevent.description }}<br/>{% endif %}
     {% for performer in subevent.performer %}{{ performer.givenName }} {{ performer.additionalName }} {{ performer.familyName }}, <span class="ome-green italic">{{ performer.affiliation.name }}</span><br/>{% endfor %}
     </p>
-  </div>
+    <ul text-align="left">
+    {% for evt in subevent.subEvent %}
+      <li>
+      <a name="{{ subevent.name }}"></a><h5>{{ evt.name }}</h5>
```

Comment:
Interestingly on Desktop this creates an unwanted extra line
![Screenshot 2021-06-02 at 08 57 32](https://user-images.githubusercontent.com/1355463/120445567-a536e300-c380-11eb-968f-34952911a573.png)
but it looks much better on mobile.

Is the anchor necessary at all as they will all likely have the same name since they are linked to the parent `subevent` name?
If we need it, could you try moving the `<a>` element inside the `<h5>`?

Reply:
Which browser? I don't see the problem.I am using Chrome

Reply:
Interesting, I cannot reproduce on Chrome either. The issue shows up on Firefox